### PR TITLE
[TRAFODION-3156] Add *Syntax Descriptions and Examples* for *EXTRACT Function* in *Trafodion SQL Reference Manual*

### PR DESCRIPTION
**Pull request by liuyu000**

### CODE REVIEW

File: docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc [hunk 1]
```diff
@@ -4437,35 +4437,670 @@ interval value expression. It returns an exact numeric value.
...
+a| * `_datetime-expression_` +
+** TIME
+** TIMESTAMP
+* `_interval-expression_` +
+
+| WEEK
+a| * `_datetime-expression_` +
+a| Week of year. +
+
+* The value 1 is returned for datetime that occurs in the first 7 days of the year if the year begins on Sunday. +
```
Comment (DaveBirdsall) [inline docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc]: Minor wordsmith. Should say, "The value 1 is returned for a datetime ... " (add an indefinite article before "datetime"). The next few sentences need this too.

Comment (liuyu000) [reply docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc]: Thanks Dave, I've corrected all 😁

**Comment by zellerh** [comment]
+1 looks good to me as well

**Comment by liuyu000** [comment]
@zellerh Nochmals herzlichen Dank 😁

**Comment by liuyu000** [comment]
@DaveBirdsall Thanks, I'll merge this PR and update Trafodion Official Website 😁

**Pull request merged**